### PR TITLE
added news page and changed the layout of blog page #6420

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by GauravButola** [comment]
LGTM

File: theme/templates/article.html [hunk 1]
```diff
@@ -1,61 +1,66 @@
...
-            </li>
-            <li class="list-unstyled post-event pull-left">
-              <a class="post-category common-link common-color" href="{{ SITEURL }}/{{ article.category.url }}">{{ article.category }}</a>
-            </li>
-          </ul>
+{% block main %}
+<section class="app-banner">
+  <div class="container">
+    <div class="text-center section-space">
+      <h1 class="white-text-color heading">{{ article.title }}
```
Comment (tarunbhardwaj) [inline theme/templates/article.html]: h1 close ?

File: theme/templates/article.html [hunk 2]
```diff
@@ -1,61 +1,66 @@
...
+</section>
+<section class="section-space">
+  <div class="container">
+    <div class="col-xs-12 col-sm-10 col-sm-offset-1">
+      <div class="col-xs-12 col-sm-12">
+        {% if article.bannerimage  %}
+          <div class="row">
+            <div class="col-md-8">
+              <div class="img-frame">
+                <a href="{{ SITE_ROOT }}{{ article.url }}">
```
Comment (tarunbhardwaj) [inline theme/templates/article.html]: not needed

File: theme/templates/base.html [hunk 3]
```diff
@@ -108,7 +108,7 @@ <h4 class="white-text-color text-center">{% trans %}Let us demonstrate how it ca
       <div class="col-md-3 col-sm-6 col-xs-12 text-center-xs footer-set-height">
         <img src="{{ STATIC }}images/footer-logo.png" alt="trytonLogo" class="img img-responsive content-center-xs" />
         <ul class="contact-address list-unstyled">
-          <li>tryton.us{% trans %} is an initiative of Openlabs Technologies to aware people about Tryton and its implementation.<br><br>It is a private initiative not intended to replace, or communicate on behalf of the whole Tryton community.{% endtrans %}
+          <li>{% trans %}tryton.us is an initiative of Openlabs Technologies to aware people about Tryton ERP and its implementation.<br><br>It is a private initiative not intended to replace, or communicate on behalf of the whole Tryton community.{% endtrans %}
```
Comment (tarunbhardwaj) [inline theme/templates/base.html]: move tryton.us out of trans

File: theme/templates/index.html [hunk 4]
```diff
@@ -29,54 +29,42 @@ <h1 class="white-text-color heading">{% trans %}Blog{% endtrans %}
...
+    <div class="col-md-10 col-md-offset-1">
+      <h3>
+        <a href="{{ SITE_ROOT }}{{ article.url }}" class="text-muted">
+          <strong>{{ article.title }} </strong>
+        </a>
+      </h3>
+      <img src="{{ STATIC }}images/article/calendar-icon.jpg" alt="calender" class="calender-icon"></img> {{ article.date|strftime('%a') }}, 
+      {{ article.date|strftime('%B') }} 
+      {{ article.date|strftime('%d') }}, 
+      20{{ article.date|strftime('%y') }}
```
Comment (tarunbhardwaj) [inline theme/templates/index.html]: use `%Y`

File: theme/templates/orders.html [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {% extends 'base.html' %}
-{% block title %}{% trans %} Order Management App {% endtrans %}| {{ super() }} {% endblock title %}
+{% block title %}{% trans %}Order Management App{% endtrans %}| {{ super() }} {% endblock title %}
```
Comment (tarunbhardwaj) [inline theme/templates/orders.html]: `s/|/ |/`

**Pull request merged**